### ISSUE TITLE
Refactor "operaClass" code on Line 54

### ISSUE THREAD

**Issue by sahat** [bestiejs/platform.js#39]
This is totally optional; you obvious had a reason for writing code this way. I personally find it hard to follow: using a variable at the same time as you define it.

```
var operaClass = reOpera.test(operaClass = getClassOf(opera)) ? operaClass : (opera = null);
```

WebStorm marks the whole file as erroneous just because of this one error. It is not even an error, since it still works fine. It would be more clear if you break up the code into two separate lines. Use your judgment for this.

![WebStorm](http://i.imgur.com/6BHUAwM.png)

**Comment by Steinerd** [comment]
... late to the party, but a quick-fix that keeps everyone happy could be this. 

```javascript
    //noinspection JSUnresolvedVariable
    /** Opera `[[Class]]` */
    var operaClass = reOpera.test(operaClass = (isCustomContext && opera) ? opera['[[Class]]'] : getClassOf(opera))
      ? operaClass
      : (opera = null);
```
WebStorm will stop complaining and you get to keep the logic-initialization.
